dashboard/app/assets/javascripts/dialog.js.erb

```diff
@@ -32,16 +65,34 @@ function Dialog(options) {
   } else if (close) {
     modalBody.append(closeLink);
   }
+
   modalBody.append(body);
   this.div.append(modalBody).appendTo($(document.body));
 
+  if (options.autoResizeScrollableElement) {
+    this.div.addClass('auto-resize-scrollable');
+    var scrollableElement = this.div.find(options.autoResizeScrollableElement);
+    scrollableElement.css('max-height', '560px');
+    scrollableElement.css('overflow-y', 'auto');
```

Comment:
I'm a little confused about this; why we need to only set these properties when using markdown, why we define these css properties here instead of attaching a class, etc.  Is this future-proofing against something?